### PR TITLE
Updates ARM template files - Blockchain

### PR DESCRIPTION
**Pull request by davidsmatlak**
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [x] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

- Updates template used in ARM template [quickstart](https://docs.microsoft.com/azure/blockchain/service/create-member-template) article.
- Adds quickstart article's author to `docOwner` in _metadata.json_.
- Updates _README.md_ with link to the quickstart article.
- Fixes order of template's `resources` properties based on best practices guidance.
- Fixes indentation.

### CODE REVIEW

**Comment by bmoore-msft** [comment]
@PatAltimore - check this PR for updates that may be needed to documentation that references this sample.

File: 201-blockchain-asaservice/README.md [hunk 1]
```diff
@@ -26,13 +23,12 @@ This template deploys a **Azure Blockchain Service**. Azure Blockchain Service i
 
 If you are new to BlockChain Services, see:
```
Comment (PatAltimore) [inline 201-blockchain-asaservice/README.md:24]: I'd change this to "If you are new to Azure Blockchain Service, see:"

File: 201-blockchain-asaservice/README.md [hunk 2]
```diff
@@ -26,13 +23,12 @@ This template deploys a **Azure Blockchain Service**. Azure Blockchain Service i
...
 
-- [Azure BlockChain Services](https://azure.microsoft.com/services/blockchain-service/).
```
Comment (PatAltimore) [inline 201-blockchain-asaservice/README.md:29]: The marketing name is Azure Blockchain Service

**Review by PatAltimore — CHANGES_REQUESTED** [review]
Suggestion to fix the service name and capitalization.

**Comment by PatAltimore** [comment]
@bmoore-msft I'll take a look.

File: 201-blockchain-asaservice/README.md (hunk 1 above)
Comment (davidsmatlak) [reply 201-blockchain-asaservice/README.md:24]: Thanks Pat, I fixed it.

File: 201-blockchain-asaservice/README.md (hunk 2 above)
Comment (davidsmatlak) [reply 201-blockchain-asaservice/README.md:29]: Thanks Pat, I fixed it.

**Comment by bmoore-msft** [comment]
@PatAltimore - check this PR for updates that may be needed to documentation that references this sample.

**Comment by PatAltimore** [comment]
Hi @bmoore-msft and @davidsmatlak 

I have PR ready with the changes https://github.com/MicrosoftDocs/azure-docs-pr/pull/130193.  I can merge when you are ready.

Pat

**Pull request merged**